### PR TITLE
Add README.md screenshots

### PR DESCRIPTION
**Pull request by lutzroeder**
* Add screenshots
* Update build status badges
* Add blog link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
-# Introducing .NET Multi-platform App UI (MAUI)
+# .NET Multi-platform App UI (.NET MAUI)
 
-.NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating native mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, macOS, and Windows from a single shared code-base. [Read More](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui)
+[![Build Status](https://dev.azure.com/xamarin/public/_apis/build/status/MAUI-public?repoName=dotnet%2Fmaui&branchName=main&label=Public)](https://dev.azure.com/xamarin/public/_build/latest?definitionId=57&repoName=dotnet%2Fmaui&branchName=main)[![Build Status](https://devdiv.visualstudio.com/DevDiv/_apis/build/status/MAUI?repoName=dotnet%2Fmaui&branchName=main&label=Private)](https://devdiv.visualstudio.com/DevDiv/_build/latest?definitionId=13330&repoName=dotnet%2Fmaui&branchName=main)
+
+.NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating native mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, macOS, and Windows from a single shared code-base.
```
Comment (jamesmontemagno) [inline README.md:5]:
```suggestion
.NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating native mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on iOS, iPadOS, Android, macOS, and Windows from a single shared code-base.
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 
 ## Getting Started ##
 
 * [Install .NET MAUI](https://github.com/dotnet/maui/wiki/Getting-Started)
```
Comment (jamesmontemagno) [inline README.md:9]:
```suggestion
* [Install .NET MAUI](https://docs.microsoft.com/dotnet/maui/get-started/installation)
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
-* [MAUI documentation](https://docs.microsoft.com/en-us/dotnet/maui)
-* [MAUI samples](https://github.com/dotnet/maui-samples)
+* [.NET MAUI documentation](https://docs.microsoft.com/en-us/dotnet/maui)
```
Comment (jamesmontemagno) [inline README.md:10]:
```suggestion
* [.NET MAUI documentation](https://docs.microsoft.com/dotnet/maui)
```

File: README.md [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
+* [.NET MAUI samples](https://github.com/dotnet/maui-samples)
 * [Developer Guide](DEVELOPMENT.md)
 
-## Build Status ##
+Follow the [.NET MAUI Blog](https://devblogs.microsoft.com/dotnet/category/net-maui/) for news and updates.
+
+## Overview
+
+.NET MAUI is open-source and is the evolution of Xamarin.Forms, extended from mobile to desktop scenarios, with UI controls rebuilt from the ground up for performance and extensibility. If you've previously used Xamarin.Forms to build cross-platform user interfaces, you'll notice many similarities with .NET MAUI. However, there are also some differences. Using .NET MAUI, you can create multi-platform apps using a single project, but you can add platform-specific source code and resources if necessary. One of the key aims of .NET MAUI is to enable you to implement as much of your app logic and UI layout as possible in a single code-base. [Read More](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui)
```
Comment (jamesmontemagno) [inline README.md:18]:
```suggestion
.NET MAUI is open-source and is the evolution of Xamarin.Forms, extended from mobile to desktop scenarios, with UI controls rebuilt from the ground up for performance and extensibility. If you've previously used Xamarin.Forms to build cross-platform user interfaces, you'll notice many similarities with .NET MAUI. However, there are also some differences. Using .NET MAUI, you can create multi-platform apps using a single project, but you can add platform-specific source code and resources if necessary. One of the key aims of .NET MAUI is to enable you to implement as much of your app logic and UI layout as possible in a single code-base. [Read More](https://docs.microsoft.com/dotnet/maui/what-is-maui)
```

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]
Made a few tweaks and removed all en-us

File: README.md (hunk 2 above)
Comment (lutzroeder) [reply README.md:9]: GitHub has some extra steps on creating first project that would have to move to the other page. Both are temporary.

**Review by jamesmontemagno — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
+.NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating native mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, iPadOS, macOS, and Windows from a single shared code-base.
 
 ## Getting Started ##
 
 * [Install .NET MAUI](https://github.com/dotnet/maui/wiki/Getting-Started)
-* [MAUI documentation](https://docs.microsoft.com/en-us/dotnet/maui)
-* [MAUI samples](https://github.com/dotnet/maui-samples)
+* [.NET MAUI documentation](https://docs.microsoft.com/dotnet/maui)
+* [.NET MAUI samples](https://github.com/dotnet/maui-samples)
 * [Developer Guide](DEVELOPMENT.md)
```
Comment (jamesmontemagno) [inline README.md:12]:
```suggestion
* [Contributor Guide](DEVELOPMENT.md)
```

Comment (jamesmontemagno) [reply README.md:12]: I think makes more sense?

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]
Thoughts?

**Review by davidortinau — APPROVED** [review]

**Pull request merged**